frontend/src/components/ViewConflict.tsx

```diff
@@ -27,29 +18,16 @@ const MainContainer = styled.div`
 `
 
 export default function ViewConflict() {
-  const dispatch = useDispatch()
-  const params = useParams<{ facilityID: string }>()
-  const { ViewStartDate, ViewEndDate, isLoading, selectedFacilityId } = useSelector(
-    (state: RootState) => state.facilityBooking,
-  )
-  useEffect(() => {
-    dispatch(SetIsLoading(true))
-    dispatch(fetchFacilityNameFromID(parseInt(params.facilityID)))
-    dispatch(getAllBookingsForFacility(ViewStartDate, ViewEndDate, parseInt(params.facilityID)))
-    if (selectedFacilityId === 0) {
-      dispatch(setSelectedFacility(parseInt(params.facilityID)))
-    }
-    return () => {
-      dispatch(setViewDates(new Date(), parseInt(params.facilityID)))
-    }
-  }, [])
+  const history = useHistory()
+  const { isLoading, conflictBookings } = useSelector((state: RootState) => state.facilityBooking)
+  if (conflictBookings.length === 0) {
```

Comment:
might want to consider doing this in useeffect so it will be dependent on conflictbookings

Reply:
Resolved in commit 83463a257bc54b74c08b93e570592ad66c382bd7.